From e2fd10bcb5619ffb56bc8aeb45e6418be5d8fbcf Mon Sep 17 00:00:00 2001
From: Elaine Zhang <zhangqing@rock-chips.com>
Date: Tue, 24 Sep 2019 17:29:59 +0800
Subject: [PATCH] clk: rockchip: rk3288: mark pclk_peri as critical clk

Avoid disable the parent clock after the child clock has disabled.

Change-Id: I1ea91afe0b6bbefd3a5d8e88641e4a3af5a368a4
Signed-off-by: Elaine Zhang <zhangqing@rock-chips.com>
---
 drivers/clk/rockchip/clk-rk3288.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/clk/rockchip/clk-rk3288.c b/drivers/clk/rockchip/clk-rk3288.c
index a32e75ed4207..59c79369cef8 100644
--- a/drivers/clk/rockchip/clk-rk3288.c
+++ b/drivers/clk/rockchip/clk-rk3288.c
@@ -822,6 +822,7 @@ static const char *const rk3288_critical_clocks[] __initconst = {
 	"aclk_rga_niu",
 	"hclk_peri",
 	"hclk_vio_niu",
+	"pclk_peri",
 	"pclk_alive_niu",
 	"pclk_pd_pmu",
 	"pclk_pmu_niu",
-- 
2.35.3

